netci.groovy

```diff
@@ -3572,6 +3572,15 @@ def static shouldGenerateFlowJob(def scenario, def isPR, def architecture, def c
         }
     }
 
+    // Disable flow jobs for innerloop pr.
+    //
+    // The only exception is windows arm(64)
+    if (isInnerloopTestScenario(scenario) && isPR && os != 'Windows_NT') {
```

Comment:
Since it's only Windows arm(64) should we check for the right `architecture` here?

Reply:
There will be an assert for windows arm and arm64 in 20 lines if it is windows.

Reply:
Can you explain why Windows_NT is exempt?

Reply:
The windows arm and arm64 jobs will remain, as we currently do not have a mechanism similar to the whitelist that would keep these machines from being overloaded on Azure Dev Ops